catalog/transformer/catalog-transformer-pdf/src/main/java/ddf/catalog/transformer/input/pdf/PdfInputTransformer.java

```diff
@@ -219,59 +170,54 @@ private Metacard transformPdf(String id, PDDocument pdfDocument, String contentI
      * @param metacard    A mutable metacard to add the extracted data to
      */
     private void extractPdfMetadata(PDDocument pdfDocument, MetacardImpl metacard) {
-        StringBuilder metadataField = new StringBuilder("<pdf>");
-
-        PDMetadata documentCatalogMetadata = pdfDocument.getDocumentCatalog()
-                .getMetadata();
-        if (documentCatalogMetadata != null) {
-            try (InputStream inputStream = documentCatalogMetadata.createInputStream()) {
-                metadataField.append(IOUtils.toString(inputStream));
-            } catch (IOException e) {
-                LOGGER.warn("Couldn't read the PDF document catalog's metadata", e);
-            }
-        }
 
         PDDocumentInformation documentInformation = pdfDocument.getDocumentInformation();
 
         Calendar creationDate = documentInformation.getCreationDate();
         if (creationDate != null) {
             metacard.setCreatedDate(creationDate.getTime());
-            addXmlElement("creationDate", DATE_FORMAT.format(creationDate), metadataField);
+            metacard.setAttribute(Core.METACARD_CREATED, creationDate.getTime());
             LOGGER.debug("PDF Creation date was: {}", DATE_FORMAT.format(creationDate));
         }
 
         Calendar modificationDate = documentInformation.getModificationDate();
         if (modificationDate != null) {
             metacard.setModifiedDate(modificationDate.getTime());
-            addXmlElement("modificationDate", DATE_FORMAT.format(modificationDate), metadataField);
+            metacard.setAttribute(Core.METACARD_MODIFIED, modificationDate.getTime());
             LOGGER.debug("PDF Modification date was: {}", DATE_FORMAT.format(modificationDate));
         }
 
-        String title = documentInformation.getTitle();
-        if (StringUtils.isNotBlank(title)) {
-            metacard.setTitle(title);
-            addXmlElement("title", title, metadataField);
+        if (isMetadataTitleTheMetacardTitle) {
+            String title = documentInformation.getTitle();
+            if (StringUtils.isNotBlank(title)) {
+                metacard.setTitle(title);
+            }
         }
 
-        addXmlElement("author", documentInformation.getAuthor(), metadataField);
-        addXmlElement("creator", documentInformation.getCreator(), metadataField);
-        addXmlElement("keywords", documentInformation.getKeywords(), metadataField);
-        addXmlElement("producer", documentInformation.getProducer(), metadataField);
-        addXmlElement("subject", documentInformation.getSubject(), metadataField);
-        addXmlElement("pageCount", String.valueOf(pdfDocument.getNumberOfPages()), metadataField);
+        String author = documentInformation.getAuthor();
+        if (StringUtils.isNotBlank(author)) {
+            metacard.setAttribute(Core.METACARD_OWNER, author);
```

Comment:
we shouldn't be setting Core.METACARD_OWNER (we need to make a note of this in the javadoc/wiki)